EventsExpress/ClientApp/src/components/privacy/privacy.css

```diff
@@ -0,0 +1,32 @@
+.privacy{
+   margin:20px 0; 
+   font-size: 18px;
+}
+.privacy p{
```

Comment:
Combine multiple similar styles to a sinlge selector or class please. And format the code if possible - alt+shift+h in vs code.